### ISSUE TITLE
A11y issue with text color of positioned badges

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/Orange-Boosted-Bootstrap#1631]
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

[Positioned badge](https://boosted.orange.com/docs/components/badge/#positioned) used in global headers has a white text on blue background.

This error was spotted by pa11y-ci and in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1099#issuecomment-1078811343.

We added a rule in `build/.pa11yci.json` to hide this warning: `.badge.rounded-pill.bg-info.text-white`.

This issue should be fixed in this order:
1. The DSM should be updated by the designers to change the color (on hold)
2. A PR must be created to:
  * Remove the `.badge.rounded-pill.bg-info.text-white` rule from `build/.pa11yci.json`
  * Apply the right color in  [Positioned badge](https://boosted.orange.com/docs/components/badge/#positioned) but also in the global headers examples everywhere

### Reduced test cases

* [Positioned badge](https://boosted.orange.com/docs/components/badge/#positioned)
* [Global headers example](https://boosted.orange.com/docs/examples/navbars/)

### What operating system(s) are you seeing the problem on?

Windows, macOS, Android, iOS, Linux

### What browser(s) are you seeing the problem on?

Chrome, Safari, Firefox, Microsoft Edge, Opera

### What version of Boosted are you using?

v5.2
